operate/services/manage.py

```diff
@@ -1059,6 +1059,76 @@ def terminate_service_on_chain(
         chain_data.on_chain_state = OnChainState.TERMINATED_BONDED
         service.store()
 
+    def reset_staking_on_chain_from_safe(  # pylint: disable=too-many-locals
```

Comment:
we mostly don't need this function, because it's basically re-calling the same functions that are present in this class (with some better UX). It makes more sense to just simplify and have it in the reset_staking.py file